File: backend/Turistando/Turistando.WebApi/Models/RoteiroCategoria.cs
namespace Turistando.WebApi.Models
{
    public class RoteiroCategoria
    {
        public RoteiroCategoria()
        {
            
        }

        public int IdRoteiro { get; set; }
        public int IdCategoria { get; set; }

        public virtual Roteiro Roteiro { get; set; }
        public virtual Categoria Categoria { get; set; }
    }
}

File: frontend/turistando-react/src/models/roteiro-categoria.ts
import {Roteiro} from "./roteiro";
import {Categoria} from "./categoria";

export class RoteiroCategoria
{
    constructor(idRoteiro: number, idCategoria: number)
    {
        this.IdRoteiro = idRoteiro;
        this.IdCategoria = idCategoria;
    }

    public IdRoteiro: number;
    public IdCategoria: number;

    public Roteiro: Roteiro;
    public Categoria: Categoria;
}
